### PR TITLE
chore(website): Add documentation to learn page

### PR DESCRIPTION
**Pull request by FuzzB0t**

### CODE REVIEW

File: packages/website/src/features/Docs/DocsCannonfilesPage.tsx [hunk 1]
```diff
@@ -395,7 +395,15 @@ export const DocsCannonfilesPage: FC = () => {
...
                   </Link>
                 </Heading>
-                <Text color="gray.400">Coming soon.</Text>
+                <Text color="gray.400">
+                  Primary source of cannon package information which contains
+                  package definition and on-chain state data derived changes
+                  made by defined steps in the cannonfile definition. Deployment
+                  data is stored on IPFS and is locally stored in your
+                  filesystem in the default storage location
+                  ~/.local/share/cannon/tags or the storage location defined by
```
Comment (mjlescano) [inline packages/website/src/features/Docs/DocsCannonfilesPage.tsx:404]: Some code styles for legibility:
```suggestion
                  <Code>~/.local/share/cannon/tags</Code> or the storage location defined by
```

File: packages/website/src/features/Docs/DocsCannonfilesPage.tsx [hunk 2]
```diff
@@ -427,7 +438,20 @@ export const DocsCannonfilesPage: FC = () => {
...
+                  package. Currently metadata includes the following:
+                  <ul>- Git Repo URL</ul>
+                  <ul>
+                    - Commit hash of the changes in which the last instance of
+                    the package were made
+                  </ul>
+                  <ul>- Link to the package git repo README file</ul>
+                  Metadata is also stored on IPFS and is locally stored in your
+                  filesystem in the default storage location
+                  ~/.local/share/cannon/tags or the storage location defined by
```
Comment (mjlescano) [inline packages/website/src/features/Docs/DocsCannonfilesPage.tsx:452]: Also here:
```suggestion
                  <Code>~/.local/share/cannon/tags</Code> or the storage location defined by
```

File: packages/website/src/features/Docs/DocsCannonfilesPage.tsx [hunk 3]
```diff
@@ -411,7 +419,10 @@ export const DocsCannonfilesPage: FC = () => {
                     #
                   </Link>
                 </Heading>
-                <Text color="gray.400">Coming soon.</Text>
+                <Text color="gray.400">
+                  Contains artifact data and other contract source code data
+                  about the contracts deployed during the build.
```
Comment (dbeal-eth) [inline packages/website/src/features/Docs/DocsCannonfilesPage.tsx:424]: We really need to expand on this with detailed information of what is *inside* the package code. Would be good to show the contents/explanation of the fields inside the data, and how to pull/parse it out.

File: packages/website/src/features/Docs/DocsCannonfilesPage.tsx [hunk 4]
```diff
@@ -427,7 +438,20 @@ export const DocsCannonfilesPage: FC = () => {
                     #
                   </Link>
                 </Heading>
-                <Text color="gray.400">Coming soon.</Text>
+                <Text color="gray.400">
+                  Metadata contains external information related to the cannon
+                  package. Currently metadata includes the following:
+                  <ul>- Git Repo URL</ul>
```
Comment (dbeal-eth) [inline packages/website/src/features/Docs/DocsCannonfilesPage.tsx:444]: this HTML does not appear correct...? how does it look in your browser? normally lists are represented by the format

```
<ul>
<li>One list item</li>
<li>Two list item</li>
</ul>
```

**Review by dbeal-eth — CHANGES_REQUESTED** [review]

File: packages/website/src/features/Docs/DocsCannonfilesPage.tsx (hunk 4 above)
Comment (noahlitvin) [reply packages/website/src/features/Docs/DocsCannonfilesPage.tsx:444]: There are chakra components for lists.

**Pull request merged**